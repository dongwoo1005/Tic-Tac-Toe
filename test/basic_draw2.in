game basic_drawX.txt basic_drawO.txt
quit
