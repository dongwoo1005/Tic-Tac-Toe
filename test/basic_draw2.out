X's move
(plays C)
O's move
(plays W)
X's move
(plays NW)
O's move
(plays SE)
X's move
(plays S)
O's move
(plays N)
X's move
(plays SW)
O's move
(plays NE)
X's move
(plays E)
Draw
Score is
X 0
O 0
